src/conspiracies/docprocessing/docprocessor.py

```diff
@@ -11,9 +11,9 @@
 
 class DocProcessor:
     def _build_coref_pipeline(self):
-        nlp_coref = spacy.blank("da")
+        nlp_coref = spacy.blank(self.language)
         nlp_coref.add_pipe("sentencizer")
-        nlp_coref.add_pipe("allennlp_coref")
+        nlp_coref.add_pipe("allennlp_coref", config={"language": self.language})
```

Comment:
Why have it as multiple pipelines (instead of just one?)

Reply:
Mimicking how I could see that you and/or others had done in the past. But I had the same question. It's up for revision.

Reply:
You at least perform the first step of the pipeline twice (but otherwise not a big thing)